### DISCUSSION TITLE
Documentation migration job

### DISCUSSION THREAD

**Discussion by jofftiquez** [jofftiquez/vue-stripe-checkout#261] [General]
GitBook gave VueStripe a Community Plan access and I decided to migrate all the docs to GitBook for better maintenance and easier management. Anyone interested in helping me migrate the docs is welcome to join. Please feel free to reach out via email through jofftiquez@gmail.com, thank you.

Current progress so far: https://docs.vuestripe.com/vue-stripe/

**Comment by britisharmy** [comment]
Website is often offline.

**Reply by jofftiquez** [reply]
Apologies, it was due to the unupdated DNS setup. Should be fine now.
